docs/FeatureManagement/Samples/NoFilters.tests.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "Test ID": "NoFilters1",
+        "Feature Flag Name": "BooleanTrueFeatureFlag",
+        "Expected Result": true,
+        "Description": "An Enabled Feature Flag with no Filters"
+    },
+    {
+        "Test ID": "NoFilters2",
+        "Feature Flag Name": "BooleanFalseFeatureFlag",
+        "Expected Result": false,
```

Comment:
We have `Inputs` property in TargetingFilter.test. Would it be better to have an empty `Inputs` for all the other tests?